templates/repo/create.tmpl

```diff
@@ -44,6 +44,8 @@
 					<div class="inline required field {{if .Err_RepoName}}error{{end}}">
 						<label for="repo_name">{{ctx.Locale.Tr "repo.repo_name"}}</label>
 						<input id="repo_name" name="repo_name" value="{{.repo_name}}" autofocus required maxlength="100">
+						<span id="repo_name_public_profile_hint" style="display:none" class="help">{{ctx.Locale.Tr "repo.repo_name_public_profile_hint"}}</span>
```

Comment:
Use `tw-hidden` class rather than `style="display:none"`.

Reply:
fixed in https://github.com/go-gitea/gitea/pull/32872/commits/84ef52704a1fab080e120181cc750c2d336d1732